package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "vanilla-framework",
-  "version": "4.16.0",
+  "version": "4.16.1",
```

`4.16.0` hasn't been released yet, so we can use it here instead of `4.16.1`

```suggestion
  "version": "4.16.0",
```